src/amo/components/App/index.js

```diff
@@ -31,7 +31,7 @@ import {
   VIEW_CONTEXT_HOME,
   maximumSetTimeoutDelay,
 } from 'core/constants';
-import DefaultErrorPage from 'core/components/ErrorPage';
+import DefaultErrorPage from 'core/components/pages/ErrorPage';
```

Comment:
I don't think `ErrorPage` makes sense in `pages`. I know that there are routes for them but they are just debugging routes used for development. I see `NotAuthorized`, `NotFound`, and so on as low-level components that routed pages will import and render.